lib/omniauth/openid_connect/util.rb

```diff
@@ -0,0 +1,47 @@
+# -*- coding:utf-8 -*-
+
+module OmniAuth
+  module OpenIDConnect
+
+    # @param [String or Hash] key_or_hash  PEM形式の証明書データ
+    # @raise [OpenSSL::X509::CertificateError] 証明書のフォーマットが不正
+    def self.parse_x509_key key_or_hash, kid
```

Comment:
I'm surprised we don't have any Rubocop enforcement of consistent styles here. I find it difficult to read function definitions without parens:

```suggestion
    def self.parse_x509_key(key_or_hash, kid)
```